store/tikv/lock_resolver.go

```diff
@@ -109,6 +109,7 @@ func NewLockResolver(etcdAddrs []string, security config.Security) (*LockResolve
 type TxnStatus struct {
 	ttl      uint64
 	commitTS uint64
+	action   kvrpcpb.Action
 }
```

Comment:
Can this definition be moved to a better place in `store` directory, so those `CheckTxnStatus` functions can return this struct instead of returning its three fields in a long tuple?

Reply:
That would involve too many irrelevant changes in this PR